lib/Service/ImportService.php

```diff
@@ -269,6 +281,35 @@ public function import(?int $tableId, ?int $viewId, string $path, bool $createMi
 			throw new InternalError($error);
 		}
 
+		$this->jobList->add(
+			ImportTableJob::class,
+			[
+				'user_id' => $this->userId,
+				'table_id' => $this->tableId,
+				'view_id' => $this->viewId,
+				'path' => $path,
```

Comment:
please suggest how to deal with files that are uploaded from a browser (not selected from user's files in Nextcloud). Those files stored in `/tmp` until PHP finishes request. Should I use `OCP\ITempManager` for that?

Reply:
This ain`t help, the files by `OCP\ITempManager` will also be deleted with the request. 

An alternative could be using `\OCP\IAppData` which basically gives you a folder within `data/appdata_INSTANCEID/`. Bookeeping has to be done in the app logic (i.e. delete uploaded files after processing and consider the error case…).

[Deck`s FileService](https://github.com/nextcloud/deck/blob/main/lib/Service/FileService.php) could be an inspiration.